### PR TITLE
Add in affiliations #65 for @vsmalladi

### PR DESCRIPTION
**Pull request by vsmalladi**
Refs #65

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -26,6 +26,17 @@ author_info:
...
+  -
+    github: vsmalladi
+    name: Venkat S. Malladi
+    initials: VSM
+    orcid: 0000-0002-0144-0564
+    twitter: katatonikat
+    email: vsmalladi@gmail.com
+    affliations:
+      - Department of Bioinformatics, University of Texas Southwestern Medical Center, Dallas, Texas, USA
+      - Bioinformatics Core Facility, University of Texas Southwestern Medicl Center, Dallas, Texas, USA
```
Comment (agitter) [inline content/metadata.yaml]: There is a typo in `Medicl`, and you can remove the city, state, and country for now.

**Comment by vsmalladi** [comment]
@agitter fixed the typos

**Review by dhimmel — APPROVED** [review]

File: content/metadata.yaml [hunk 2]
```diff
@@ -26,6 +26,17 @@ author_info:
     email: slochower@gmail.com
     affiliations:
       - Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego
+  -
+    github: vsmalladi
+    name: Venkat S. Malladi
+    initials: VSM
+    orcid: 0000-0002-0144-0564
+    twitter: katatonikat
```
Comment (agitter) [inline content/metadata.yaml:34]: @vsmalladi is this a typo?  Should it be `katatonikkat`?

Comment (vsmalladi) [reply content/metadata.yaml:34]: Yes it should be katatonikkat.
 Also affiliations is spelt wrong

**Pull request merged**